### ISSUE TITLE
Errors in Pubmatic adapter after #453

### ISSUE THREAD

**Issue by dmitriyshashkin** [prebid/Prebid.js#538]
Since #453 I started seeing lots of errors from Pubmatic adapter. 

  TypeError: undefined is not an object (evaluating 'window.frames[a].contentWindow.bidDetailsMap')

It seems that window.frames[iframeId].contentWindow does not work in Safari (both desktop and mobile). Because according to MDN window.frames returns window object, not a frame element.

My proposal would be to save a reference to frame object in a variable and don't use window.frames property at all. That fixed the problem for me. Should I submit a pull request?

**Comment by mkendall07** [comment]
@dmitriyshashkin 
if you have the fix already, go ahead and submit the PR. I can review. Thanks

**Comment by mkendall07** [comment]
@dmitriyshashkin 
is this issue only happening in Safari?

**Comment by dmitriyshashkin** [comment]
Yes, it seems that the problem exists only in Safari browser.

**Comment by dmitriyshashkin** [comment]
Created a new pull request
